code/script.js

```diff
@@ -256,70 +267,117 @@ const checkQuestion = () => {
   // See if we should keep or remove people based on that
   // Then invoke filterCharacters
   if (category === 'hair' || category === 'eyes') {
+    if (value === secret[category]) {
+      filterCharacters(true)
+      console.log("keep")
+    } else
+      filterCharacters(false)
+    console.log("remove")
 
   } else if (category === 'accessories' || category === 'other') {
-
+    if (secret[category].includes(value)) {
+      filterCharacters(true)
+      console.log("keep")
+    } else
+      filterCharacters(false)
+    console.log("remove")
```

Comment:
No need to log keep and remove, but super nice use of array notation to get the desired property!